im21door.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-bitwise */
+// This file relies heavily on the use of bitwise-AND (&) and bitwise-OR (|)
```

Comment:
I don't see any bitwise ors unless im missing something

Reply:
You're right. I took out the bitwise ORs, I'll remove it from the comments.

Reply:
Done.